### PR TITLE
Adding requirements for the project

### PR DESCRIPTION
**Pull request by Martincho2003**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+asgiref==3.4.1
+Django==4.0.1
+sqlparse==0.4.2
+tzdata==2021.5
```
Comment (marto55) [inline requirements.txt:4]: Should pylance and the other extensions of vscode be listed here, excuse if my question is stupid

**Review by marto55 — COMMENTED** [review]
seems right to me
I agree

File: requirements.txt (hunk 1 above)
Comment (Martincho2003) [reply requirements.txt:4]: This mustn't be in the requerements.txt file, if u want that can be placed in the README.md to help developers work on our code, but if they use vscode they might have installed them already.

Comment (marto55) [reply requirements.txt:4]: You are right, actually vscode suggests installing those. No need to add them, You convinced me 100%

**Review by marto55 — APPROVED** [review]

**Pull request merged**